src/content/blog/dev-portal.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "The Astro Developer Portal"
+description: "Today, we’re launching the Astro Developer Portal, a platform where theme authors and creators can create, own and update their themes built for Astro."
+publishDate: "March 13, 2024"
+authors:
+  - elian
+coverImage: "/src/content/blog/_images/dev-portal/dev-portal.webp"
+socialImage: "/src/content/blog/_images/dev-portal/dev-portal.webp"
+lang: "en"
+---
+
+import BlogContentImage from "/src/components/BlogContentImage.astro"
+import BlogSummary from "/src/components/BlogSummary.astro"
+import dashboard from "/src/content/blog/_images/dev-portal/dashboard.webp"
+
+<BlogSummary>
+Today, we're launching [the Astro Developer Portal](https://portal.astro.build), a platform where theme authors and creators can create, own and update their themes built for Astro.
```

Comment:
```suggestion
Today, we're launching [the Astro Developer Portal](https://portal.astro.build), a platform where theme authors and creators can submit, manage, and promote their themes built for Astro.
```
So, they can't "create their themes" here -- this isn't a theme builder. They can upload/manage the stuff they build!

I did like the word "update" which got cut from this sentence, but I feel like that will be described in the whole process below. For here, the key thing is this is where they can *manage* it, and all that entails.